sequencing/ATACSeq/subScripts/groupPeaks.sh

```diff
@@ -45,7 +45,7 @@ echo "Starting peak calling using MACS3 PE on samples that belong to cell fracti
 date -u
 
 cd ${TMPDIR}
-macs3 callpeak -t  ${bamFiles[@]} --outdir ${PEAK_DIR}/MACS/BAMPE/group -n ${GROUP} -f BAMPE -g 2.9e9 -q 5e-2 --keep-dup all --nomodel --broad --broad-cutoff 5e-2
+macs3 callpeak -t  ${bamFiles[@]} --outdir ${PEAK_DIR}/MACS/BAMPE/group -n ${GROUP} -f BAMPE -g 2.9e9 -q 1e-3 --keep-dup all --nomodel --broad --broad-cutoff 1e-3
```

Comment:
Do we want to make the cutoff a user defined variable in the config file or does this not really matter?

Reply:
Yes, it could be a user defined option